src/services/pseudonym/index.js

```diff
@@ -15,10 +16,16 @@ module.exports = function () {
 		},
 		lean: false,
 	};
-
+	app.use('/pseudonymModel', service({ ...options, lean: true }));
```

Comment:
If you replace in testObjects the model service with the model (is used in the most helpers), then this services can removed. Many thanks for improving the testObject helpers.

Reply:
Done